l1-contracts/deploy-scripts/DeployL1.s.sol

```diff
@@ -139,6 +140,8 @@ contract DeployL1Script is Script, DeployUtils {
             "MessageRoot"
         );
 
+        deployServerNotifier();
```

Comment:
please use deployTuppWithContract, it is much more simple :)

Reply:
You need to put the creationCode, creationData and initializeData in the correct functions

Reply:
fixed